### PR TITLE
re-arrange proposed video and content into two columns

### PR DESCRIPTION
**Pull request by insectengine**
Adding Emmanuel's presentation video helps add historical context, but having it full width visually dominates all other content on the page. I suggest breaking the text and the video into columns to reduce this effect.

### CODE REVIEW

**Comment by insectengine** [comment]
Alternative to https://github.com/quarkusio/quarkusio.github.io/pull/1784

**Comment by maxandersen** [comment]
would it make sense to center the text and video rather than split in half?

**Comment by insectengine** [comment]
Nothing else is centered on the page, would seem out of place and awkward. We could flip the video and text boxes to put the whitespace on the right.
Screenshot 2024-07-10 at 10 44 44 AM

File: _includes/about.html [hunk 1]
```diff
@@ -9,29 +9,36 @@ <h2>Open Source Utterly and Absolutely</h2>
       <h2>What Makes Quarkus Different?</h2>
     </div>
     <div class="width-3-12 width-12-12-m">
-      <img class="light-only" src="{{site.baseurl}}/assets/images/about/icon-developerjoy.svg">
-      <img class="dark-only" src="{{site.baseurl}}/assets/images/about/icon-developerjoy-dark.svg">
+      <img src="{{site.baseurl}}/assets/images/about/icon-developerjoy.svg">
```
[the remark this answers is not in the document]
Comment (insectengine) [reply _includes/about.html]: Weird. Not sure how they were lost. I've updated.

[the remark this answers is not in the document]
Comment (gastaldi) [reply _includes/about.html]: Hm, looks this PR now has content from https://github.com/quarkusio/quarkusio.github.io/pull/2080?

**Comment by cescoffier** [comment]
@insectengine can you rebase to we can merge it?

**Review by cescoffier — APPROVED** [review]

**Pull request merged**